mmdet/core/loss/losses.py

```diff
@@ -56,6 +56,8 @@ def weighted_sigmoid_focal_loss(pred,
 def mask_cross_entropy(pred, target, label):
     num_rois = pred.size()[0]
     inds = torch.arange(0, num_rois, dtype=torch.long, device=pred.device)
+    if pred.shape[1] == 1:
+        label.zero_()
```

Comment:
We could avoid the inplace modification of `label` in case that it will be used afterwards.